tracing/tracing.go

```diff
@@ -146,3 +152,30 @@ func LogKV(k, v string, ctx context.Context) {
 		span.LogKV(k, v)
 	}
 }
+
+// GetTraceID retrieves TraceID from HTTP request, for example to search for this trace
+// in the UI of your tracing solution and to get more context about it
+func GetTraceID(req *http.Request) string {
+	span := ot.SpanFromContext(req.Context())
+	if span == nil {
+		return ""
+	}
+
+	spanContext := span.Context()
+	if spanContext == nil {
+		return ""
+	}
+
+	switch spanContextType := spanContext.(type) {
+	case origbasic.SpanContext:
+		return fmt.Sprintf("%x", spanContextType.TraceID)
+	case originstana.SpanContext:
+		return fmt.Sprintf("%x", spanContextType.TraceID)
+	case origjaeger.SpanContext:
+		return fmt.Sprintf("%x", spanContextType.TraceID().String())
```

Comment:
`spanContextType.TraceID().String()` already returns hexadecimal string so the format specifier should be `%s`